ansible/roles/linux-podman/templates/systemd.pod.service.j2

```diff
@@ -1,8 +1,8 @@
 [Unit]
 Description=Podman container-{{ podman_service_name }}.service
 Documentation=man:podman-generate-systemd(1)
-Wants=network.target
-After=network-online.target
+Wants=network.target user@1001.service
```

Comment:
Can you get the uid by registering an output from this task?

https://github.com/stackhpc/azimuth-images/blob/b86c461b9bf79ba0407251fa103273d47f43670c/ansible/roles/linux-podman/tasks/install.yml#L43-L46

Reply:
It [looks like](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/user_module.html#return-uid) we only get the uid in the output when we pass it in the input... So maybe we have to explicitly choose the podman uid everywhere?